.github/workflows/build-and-deploy-wasm-bindings.yml

```diff
@@ -0,0 +1,72 @@
+name: Build and Publish WASM Package
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'wasm-attestation-bindings/**'
```

Comment:
we should trigger this if the root `attestation-doc-validation` project changes

Reply:
done!